CorsixTH/Lua/entities/vip.lua

```diff
@@ -265,15 +281,14 @@ function Vip:setVIPRating()
     end
   else
     local deathRatio = numberVisitorsDiff / deathDiff
-    if deathRatio > 12 then
-      self.vip_rating = self.vip_rating + 10
-    elseif deathRatio >= 8 then
-      self.vip_rating = self.vip_rating + 5
-    elseif deathRatio >= 2 and deathRatio <= 4 then
-      self.vip_rating = self.vip_rating - 10
-    elseif deathRatio < 2 then
-      self.vip_rating = self.vip_rating - 20
-    end
+    local death_ratio_table = {
```

Comment:
If we agree to call this a range map above I'd like to keep that name here - e.g. death_ratio_rangemap.